lib/providers/player/audio_player_provider.dart

```diff
@@ -19,10 +20,11 @@ final audioPlayerNotifierProvider =
 class AudioPlayerNotifier extends BaseAudioHandler
     with QueueHandler, SeekHandler, ChangeNotifier {
   var backgroundSoundAudioPlayer = AudioPlayer();
-  TrackFilesModel? currentlyPlayingTrack;
-  final hasBgSound = 'hasBgSound';
   final trackAudioPlayer = AudioPlayer();
-  final fadeDuration = 5;
+
+  TrackFilesModel? currentlyPlayingTrack;
+  final hasBgSoundId = 'hasBgSound';
```

Comment:
I think the name should be hasBgSound instead of hasBgSoundId. Lmk, what do you think?

Reply:
`hasBGSound` implies it's a boolean, but this is really a key... maybe it should be called `hasBgSoundKey`

Reply:
Also 'trackId' and 'fileId' should be constants too but I didn't do them here

Reply:
Correct **hasBgSound** should be called **hasBgSoundKey**